ripples/multi_session.py

```
@@ -25,403 +32,1150 @@
+
+        if np.logical_and(firing_around_ripple > 0, firing_during_ripple > 0):
+            modulation[cluster] = firing_during_ripple / firing_around_ripple
+        elif np.logical_and(firing_around_ripple == 0, firing_during_ripple > 0):
+            firing_around_ripple = 0.5
+            modulation[cluster] = firing_during_ripple / firing_around_ripple
+        elif np.logical_and(firing_around_ripple > 0, firing_during_ripple == 0):
+            firing_during_ripple = 0.5
+            modulation[cluster] = firing_during_ripple / firing_around_ripple
+
+        if group == "pos_responders":
+            if modulation[cluster] > 1.5:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)